src/atoms/second_order_cone/rationalnorm.jl

```diff
@@ -60,7 +50,8 @@ function rationalnorm(x::AbstractExpr, k::Rational{Int})
         if row == 1 || col == 1
             return RationalNormAtom(abs(x), k)
         end
-    else
-        return RationalNormAtom(x, k)
+        # FIXME(odow): what happens if x is a matrix?
```

Comment:
Follow-up later. The function previously returned `nothing` if `x` was a complex matrix.